src/helpers/server.js

```diff
@@ -1,39 +1,42 @@
 import { Promise } from 'es6-promise';
 import express from 'express';
 
+/** 
+ * Wrapper around setting up an express server with static middleware
+ */
 class Server {
```

Comment:
Since this exports a class, the file name should be uppercase... maybe git didn't pick up the change?

Reply:
yes, it annoyingly didn't pick it up, will force the change somehow.

Reply:
you usually need to `git rm`, `git commit` and `git add` it again in OSX, iirc